fili-core/src/main/java/com/yahoo/bard/webservice/druid/client/impl/AsyncDruidWebServiceImpl.java

```diff
@@ -97,49 +136,52 @@ public AsyncDruidWebServiceImpl(
             ObjectMapper mapper,
             Supplier<Map<String, String>> headersToAppend
     ) {
-        this(serviceConfig, initializeWebClient(serviceConfig.getTimeout()), mapper, headersToAppend);
+        this(
+                serviceConfig,
+                initializeWebClient(serviceConfig.getTimeout()),
+                mapper,
+                headersToAppend,
+                DEFAULT_JSON_NODE_BUILDER_STRATEGY
```

Comment:
JavaDoc on this should call out that this constructor uses a particular strategy as the default.